ipn/ipnlocal/c2n.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ipnlocal
+
+import (
+	"io"
+	"net/http"
+)
+
+func (b *LocalBackend) handleC2N(w http.ResponseWriter, r *http.Request) {
+	switch r.URL.Path {
+	case "/hi":
+		io.WriteString(w, "hi\n")
```

Comment:
Let's do an echo instead, it helps test payloads too

Reply:
sure